packages/atlas-react/tests/components/BasicButton.test.js

```diff
@@ -44,7 +48,9 @@ describe("BasicButton", () => {
       <BasicButton label="Click Me" htmlType="submit" htmlId="buttonWithForm" />
     );
 
-    expect(screen.getByShadowText("Click Me").getAttribute("type")).toBe("submit");
+    expect(screen.getByShadowText("Click Me").getAttribute("type")).toBe(
```

Comment:
all tests were broken.  please make sure to familiarize on how to work with turbo instead of jumping between `cd packages/...` and stuff.

here's a very simple guide on how to do some things:

```
yarn workspace @productplan/atlas test [-u]
yarn workspace @productplan/atlas add <package-name> [-D]
```

also, before committing, you can always double check if everything is building + the lintin and format passes by just executing this simple command:

```
yarn export
```

this is the same command that is executed on CI/CD.